install.sh

```diff
@@ -27,8 +27,8 @@ tar -C $INSTALLER_DIR/ -xzf $INSTALLER_DIR/osiam-dist.tgz
 
 # add configs
 mkdir /etc/osiam
-mv $OSIAM_DIST_DIR/osiam-server/osiam-resource-server/configuration/* /etc/osiam
-mv $OSIAM_DIST_DIR/osiam-server/osiam-auth-server/configuration/* /etc/osiam
+mv $OSIAM_DIST_DIR/resource-server/configuration/* /etc/osiam
+mv $OSIAM_DIST_DIR/auth-server/configuration/* /etc/osiam
```

Comment:
No need to change the paths. They stayed the same despite the split of the servers.